### PR TITLE
Make a narrower button font for cp-1251 (Cyrillic) characters

### PR DESCRIPTION
**Pull request by Districh-ru**
relates to #6335

This PR makes Cyrillic characters more narrow to be able to write long words on buttons.
изображение изображение

But some buttons are still narrow for the translated words :(
изображение

Suggestions and comments are welcome.

Language file with alphabet in first two main menu buttons: [ru.zip](https://github.com/ihhub/fheroes2/files/10501113/ru.zip)

### CODE REVIEW

**Comment by AAS** [comment]
Maybe one should use hyphenation?

```
Сохра-
нить игру
```

and

```
Загру-
зить игру
```

Or even 3 lines.

**Comment by Districh-ru** [comment]
Hi @AAS, thanks for the suggestion.
But in this PR we are only changing the look of the cp-1251 letters. Russian translation of in-game buttons is discussed in #6335 and if you have any suggestions and comments on the translation, you are welcome.

**Review by ihhub — APPROVED** [review]

File: src/fheroes2/gui/ui_font.cpp [hunk 1]
```diff
@@ -4117,46 +4124,44 @@ namespace
         released[146] = released[41];
 
         // S
-        released[157] = released[51];
+        released[157].resize( released[51].width() - 1, released[51].height() );
```
[the remark this answers is not in the document]
Comment (Districh-ru) [reply src/fheroes2/gui/ui_font.cpp:4127]: Hi, @zenseii!
Thanks for the review.
It is funny, but the screenshot, by mistake, was form the pre-final build of this commit :)
I've checked and letter `S` renders fine with all its shadows. I've updated the screenshot.
Also if you want to check the font in game you can use the attached language file with alphabet in first two buttons: [ru.zip](https://github.com/ihhub/fheroes2/files/10501113/ru.zip)
Sorry for not attaching it earlier.

**Comment by Branikolog** [comment]
Hi, @Districh-ru !
I've tried to make some letters of more consistent design to the overall style.
Could you, please, look at my proposed letters? 
В
З
Х

"З" being comparable to "C" looks a bit too wide (comparing it to all other symbols too), so I propose make it thinner. On the other hand, I couldn't provide more estetically pleasing thin version of "C", but the one from the screenshots above looks acceptable. It's all about the visual perception... "Х" looks a bit vider too here.
Also, I'm proposing other version of "В" as commonly higher part of the symbol should be smaller than lower, which is not the case for H2. :)
How do you think of it?

**Comment by Branikolog** [comment]
@Districh-ru 
Great! 
Once you add these 4 new letters, the whole alphabet would be fine!

**Comment by Districh-ru** [comment]
> @Districh-ru Great! Once you add these 4 new letters, the whole alphabet would be fine!

@Branikolog
Done. :)
I've also updated the screenshots at the PR title post.

**Review by Branikolog — APPROVED** [review]
Great job!

**Comment by ihhub** [comment]
@Districh-ru , well done! Thank you very much for this improvement!

**Pull request merged**